src/RubberDuckyParking/parking_api/models.py

```diff
@@ -56,7 +56,7 @@ class Attendant(BaseProfile):
     The person that scans in Users as they arrive. Attendant and Host might be the same person, however for larger lots
     the host could have one or many attendants that are the 'gatekeepers' for the lot.
     """
-
+    boss = models.ForeignKey(Host, on_delete=models.SET_NULL, default=None, null=True)
```

Comment:
So if an attendant links to itself as a boss, then we know that they are the base host?

Reply:
yeah so if an attendant links itself to a host, that's its host, it can't be an attendant for multiple hosts. I thought it just made things more simple, we could change it, but idk if we care enough to